src/test_l1_dsHdmiIn.c

```diff
@@ -344,7 +341,6 @@ void test_l1_dsHdmiIn_negative_dsHdmiInGetNumberOfInputs(void) {
     UT_LOG("\n Out %s\n", __FUNCTION__); 
 }
 
-
 /**
  * @brief Ensure dsHdmiInGetStatus() correctly fetches the HDMI input port status during positive scenarios.
```

Comment:
in the dsHdmiInGetStatus 
we need to have logic to check  for source devices if there is no hdmi input support
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsHdmiIn.h#L172

please check other API's also

Reply:
Done